src/templates/root.yml

```diff
@@ -87,3 +87,11 @@ configurations:
   $namespace pancakeswap:
     configurationPath: pancakeswap.yml
     schemaPath: pangolin-schema.json
+
+  $namespace injectiveCLOB:
```

Comment:
we are trying to set a standard of 1 template file per connector. Can you move the `gasLimitEstimate` variable from injectiveCLOB into the injective.yml template so that injectiveCLOB can be removed?

Reply:
i know injective is both a chain and connector in the Gateway architecture, but I think users would prefer to have all the Injective configs in one file, and not two.

Reply:
done